packages/gatsby-plugin-offline/src/gatsby-node.js

```diff
@@ -105,11 +105,10 @@ exports.onPostBuild = (args, pluginOptions) => {
     clientsClaim: true,
   }
 
-  // pluginOptions.plugins is assigned automatically when the user hasn't
-  // specified custom options - Workbox throws an error with unsupported
-  // parameters, so delete it.
-  delete pluginOptions.plugins
```

Comment:
Just noticed this. Do we have to handle it if pluginOptions.workboxConfig .plugins exists?

Reply:
The `plugins` is injected by `gatsby` as a field on `pluginOptions`. If we are moving workbox build config to be subfield - the normalization of workboxConfig is no longer needed